### ISSUE TITLE
npm install package.json dependencies of project?

### ISSUE THREAD

**Issue by err0** [redguide/nodejs#56]
If I am installing a project that has it's own package.json file what is the syntax for that?  what is the equivalent of running "npm install" from the project root with nodejs_npm?

**Comment by guilhem** [comment]
@err0 good question :)
can you past the command you want to exec?

There is an Array `option` where you can put what you want.

**Comment by err0** [comment]
@guilhem yes, but in the nodejs_npm code there is a constraint that package be true.

if I were doing it by hand I would do:
```
cd /path/to/my/project
npm install
```
in package.json there is a name attribute so I ended up using that and it worked just fine.

my project is named dashboard in the packages.json file so this is what I did (and worked):
```
nodejs_npm "dashboard" do
  path "/home/ubuntu/deployment"
  json true
end
```
in this case what I had tried first before looking in to the source was:
```
nodejs_npm do
  path "/home/ubuntu/deployment"
  json true
end
```

I will close this issue since there is a work around. Ideally though nodejs_npm command constraints would match those of npm.
